js/src/views/Account/Header/header.js

```diff
@@ -67,6 +71,10 @@ export default class Header extends Component {
             <Balance
               account={ account }
               balance={ balance } />
+            <Certifications
+              certifications={ certifications }
+              dappsUrl={ api.dappsUrl }
```

Comment:
@ngotchac Not sure I get what you mean, [but "dumb"/presentational components are usually *not* aware of Redux](http://redux.js.org/docs/basics/UsageWithReact.html#presentational-and-container-components).